docs/usage.rst

```diff
@@ -2,44 +2,27 @@
 Running a simulation
 ====================
 
-``pyuvsim`` takes a UVData object, instrument configuration settings, and a source catalog and effectively "fills" the UVData object with simulated data. The function ``run_pyuvsim`` in uvsim.py
+``pyuvsim`` takes a UVData object, instrument configuration settings, and a source catalog and effectively "fills" the UVData object with simulated data. The function ``uvsim.run_pyuvsim`` in uvsim.py accepts as input a path to a yaml ``obsparam`` file and writes out the data to ``uvfits`` format. Optionally, it can also skip writing the data out and just return a ``UVData`` object filled with simulated data.
```

Comment:
Should this be updated to indicate that miriad files can be written out too? Out of curiosity, why miriad and not uvh5? uvh5 is what HERA is using going forward, it supports both phased and drift data and has good support for partial reading and partial writing.

Reply:
I still need to add a way to specify output format from the wrapper scripts and functions (at the moment, I've just changed the `write_uvdata` function to accept those options). I'll update the docs once that's done.

I went with miriad because I've been borrowing these functions for my healpix sky simulator and need those outputs in miriad. I'll add the uvh5 option too. Is there a common extension those files take?

Reply:
yes, `.uvh5`

Reply:
Should be addressed now. Users may specify output format through the keyword `output_format` in the `filing` section of the obsparam. Default is to write to uvfits, but uvh5 and miriad are also supported.